application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -0,0 +1,184 @@
+package org.togetherjava.tjbot.commands.moderation.modmail;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.InteractionHook;
+import net.dv8tion.jda.api.interactions.commands.Command;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.requests.restaction.MessageCreateAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.utils.DiscordClientAction;
+import org.togetherjava.tjbot.config.Config;
+
+import java.awt.*;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+
+/**
+ * Implements the /modmail command, which allows users to contact a moderator within the server or
+ * even if not part of the server anymore, e.g. after a ban.
+ * <p>
+ * Therefore, the user uses the SlashCommand /modmail and the message are forwarded to moderators in
+ * a dedicated channel.
+ * <p>
+ * Messages are then forwarded to a dedicated channel, given by
+ * {@link Config#getModAuditLogChannelPattern()}.
+ */
```

Comment:
* u describe everything twice (use slash command, forward to channels. use slash command, forward to channel).
* the first paragraph still says it can be used outside of server and after ban, which is wrong now.

Reply:
Done